README.md

```diff
@@ -121,6 +121,6 @@ To build the meta-package pysal see [build.md](build.md).
 License information
 ===================
 
-See the [LICENSE](https://github.com/pysal/pysal/blob/master/LICENSE.txt) for information on the history of this
+See the [LICENSE](pysal/blob/master/LICENSE.txt) for information on the history of this
```

Comment:
@Siddharths8212376 It may be better to keep the full URL in the link, as you had done in the previous commit (3cd3acd).

Reply:
Yeah I've updated that. Thanks!